test/cases/api.sh

```diff
@@ -145,6 +145,25 @@ function instanceWaitSSH() {
   done
 }
 
+function instanceCheck() {
+  local _ssh="$1"
+
+  # Check if postgres is installed
+  $_ssh rpm -q postgresql
+
+  # Verify subscribe status
+  subscribe_org_id=$($_ssh sudo subscription-manager identity | grep 'org ID')
+  [[ "$subscribe_org_id" == "org ID: $API_TEST_SUBSCRIPTION_ORG_ID" ]]
+
+  # Remove dos2unix before install it
+  $_ssh rpm -q dos2unix && $_ssh sudo rpm -e dos2unix
+  # Verify yum install a small package
+  $_ssh sudo dnf -y install dos2unix
+
+  # Unregister subscription
+  $_ssh sudo subscription-manager unregister || true
```

Comment:
Does this fail often? Why the `|| true`?

Reply:
I've never met the failed case. Just to avoid the risk because I think it's not a checkpoint and if it's failed it should not make the whole test failed. Do you suggest to remove the || true? Thanks!

Reply:
"Need it robust, make it fragile" -> https://www.youtube.com/watch?v=WOy9zhzyMOE

IMO let this fail the test if unregister fails. Otherwise we will exhaust the pool of subscriptions and the subsequent tests will still fail but be a bit harder to debug.

Reply:
OK thanks @atodorov ! Removed it.